crowbar_framework/lib/crowbar/validator/network_validator.rb

```diff
@@ -4,7 +4,7 @@ class NetworkValidator
       def validate_network
         `#{Rails.root.join("..", "bin/network-json-validator").to_s} \
           --admin-ip \
-          #{Socket.ip_address_list.detect(&:ipv4_private?).ip_address} \
+          #{IPSocket.getaddress(Socket.gethostname)} \
```

Comment:
How does it find the IP of the hostname? If the host has several IP addresses which one is found?